drivers/sensor/vcnl4040/vcnl4040.h

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2020 Richard Osterloh
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_VCNL4040_VCNL4040_H_
+#define ZEPHYR_DRIVERS_SENSOR_VCNL4040_VCNL4040_H_
+
+#include <drivers/sensor.h>
+#include <drivers/i2c.h>
+#include <drivers/gpio.h>
+
+/* Registers all 16 bits */
+#define VCNL4040_REG_ALS_CONF		0x00
```

Comment:
`U` suffix may be used for unsigned constants

Reply:
Since they're registers and are given in hex in basically every datasheet I think it makes sense to leave them in this form